src/orb.yml.hbs

```diff
@@ -132,6 +132,31 @@ jobs:
           "The name of the service to update. If undefined, we assume `family` is the name of both the service and task definition."
         type: string
         default: ""
+      deployment-controller:
+        description:
+          "The deployment controller to use for the service. Defaulted to ECS"
+        type: string
+        default: "ECS"
+      application-name:
+        description:
+          "The name of an AWS CodeDeploy application."
+        type: string
+        default: ""
+      deployment-group-name:
+        description:
+          "The name of an AWS CodeDeploy deployment group."
+        type: string
+        default: ""
+      container-name-for-code-deploy:
+        description:
+          "The name of the container for CodeDeploy."
+        type: string
+        default: ""
+      container-port-for-code-deploy:
```

Comment:
Could this be changed to `codedeploy-load-balanced-container-port` for clarity? (Please feel free to suggest a better name!)

Reply:
Fixed by 102d851 .